pkgs/servers/web-apps/lemmy/ui.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, mkYarnPackage
+, libsass
+, nodejs
+, python3
+, pkg-config
+, writeShellScriptBin
+, jq
+, runCommand
+, fetchFromGitHub
+}:
+
+let
+  pkgConfig = {
+    node-sass = {
+      nativeBuildInputs = [ ];
+      buildInputs = [ libsass pkg-config python3 ];
+      postInstall = ''
+        LIBSASS_EXT=auto yarn --offline run build
+        rm build/config.gypi
+      '';
+
+      dontFixup = true;
+
```

Comment:
```suggestion
      dontFixup = true;
```
Why not fixup?

Reply:
There are errors on some patchelf things, we can try again and post the errors